NEWS.md

```diff
@@ -1,6 +1,11 @@
 # MpiIsoApp development version
 
-## Version 23.09.0
+## Version 23.09.1
+
+## New Features
+- [please add some description]  
```

Comment:
Can you please add some description what is new now?

Reply:
added